packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json

```diff
@@ -154,9 +154,7 @@
               }
             ]
           },
-          "linkedEntityGuids": [
-            "MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI"
-          ]
+          "linkedEntityGuids": ["MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI"]
```

Comment:
I wonder if these will work if they are imported into other accounts. cc @polfliet

Reply:
Here is the response from the Dashboard team:

> Dashcow doesn’t process the linkedEntityGuids it receives, it just stores them in the database. So if another customer imports that dashboards and they don’t have access to MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI, then they wouldn’t be able to retrieve the entity outline (name, etc) of that linked dashboard as Nataly pointed out, and facet linking won’t work.

So if I understand correctly the user would still see the dashboard and that specific widget, but would not be able to use the advanced features like facet linking (which is ok I think)

Reply:
@polfliet ok oh, I'm not super familiar with how this works but it sounds like this is 🆗